templates/common/infra/bicep/core/host/functions-dotnet.bicep

```diff
@@ -0,0 +1,36 @@
+param environmentName string
+param location string = resourceGroup().location
+
+param allowedOrigins array = []
+param applicationInsightsName string
+param appServicePlanId string
+param appSettings object = {}
+param keyVaultName string = ''
+param linuxFxVersion string = ''
+param kind string = 'functionapp'
```

Comment:
Are you sure we need to change this from 'functionapp,linux' to 'functionapp'?  function.bicep will take care of kind, so not sure if we need to override.

Reply:
If we want to target Windows we would need kind to be functionapp and not functionapp,linux. When I try to deploy an out of process to Linux comsumption it gets a deploy error that doesn't happen with Windows consumption. I haven't worked with Functions team to troubleshoot though. This is relevant https://github.com/Azure/azure-functions-dotnet-worker/issues/821

Reply:
Fixed in latest commit!